### PR TITLE
Adding raise_on_error option to cmd line

### PR DESCRIPTION
**Pull request by schrockn**
Also fixing up some error messages.

Resolves https://github.com/dagster-io/dagster/issues/637

### CODE REVIEW

File: examples/toys/repo.py [hunk 1]
```diff
@@ -1,4 +1,11 @@
+import os
+import sys
 from dagster import RepositoryDefinition
+from dagster.utils import script_relative_path
+
+sys.path.insert(0, os.path.abspath(script_relative_path('.')))
```
Comment (schrockn) [inline examples/toys/repo.py]: @mgasner this is required to run this from the command line.

@mgasner do you have a sense why the cmdline needs this but dagit does not?

Comment (mgasner) [reply examples/toys/repo.py]: what do you mean to run this from the command line? `python repo.py` works fine for me..

Comment (alangenfeld) [reply examples/toys/repo.py]: ideally we figure this out and can remove this before landing

Comment (alangenfeld) [reply examples/toys/repo.py]: alright i think i just hit this, the repro is 
`dagster pipeline execute error_monster` from inside `examples/toys`

Comment (alangenfeld) [reply examples/toys/repo.py]: https://gist.github.com/alangenfeld/9cfa3650109901a39d44fd2e942ea80f

**Pull request merged**